config/runtime/tests/v4l2-decoder-conformance.jinja2

```diff
@@ -0,0 +1,35 @@
+{%- if testsuite -%}
+    {% set testsuite_arg = "-ts " + testsuite %}
+{%- endif -%}
+{%- if decoders -%}
+    {%- set decoder_list = [ ] -%}
+    {%- for decoder in decoders -%}
+        {{ decoder_list.append(decoder) or "" }}
+    {%- endfor -%}
+    {%- set decoders_arg = "-d " + decoder_list|join(" ") -%}
+{%- endif -%}
+{%- if videodec_timeout -%}
+    {% set videodec_timeout_arg = "-t " + videodec_timeout|string %}
+{%- endif -%}
+{%- if videodec_parallel_jobs -%}
+    {% set videodec_parallel_jobs_arg = "-j " + videodec_parallel_jobs|string %}
+{%- endif -%}
+- test:
+    timeout:
+      minutes: {{ job_timeout }}
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: {{ node.name }}
+          description: "V4L2 decoder conformance test plan"
+          os:
+          - debian
+          scope:
+          - functional
+        run:
+          steps:
+          - python3 /usr/bin/fluster_parser.py {{ testsuite_arg }} {{ decoders_arg }} {{ videodec_timeout_arg }} {{ videodec_parallel_jobs_arg }}
+      from: inline
+      name: {{ plan }}
+      path: inline/{{ plan }}.yaml
```

Comment:
Where is `plan` defined? I suspect this was available in the legacy system but not something we have in the new system.

Reply:
@laura-nao @nuclearcat any ideas about this?

Reply:
Right, I missed this. `plan` doesn't seem to be used by the new system indeed, I think we can use `node.name`  instead here as well.

Reply:
Okay. Put name: node.name and removed path.

Reply:
@musamaanjum the path is a requirement for inline test definitions in LAVA (https://docs.lavasoftware.org/lava/writing-tests.html#using-inline-test-definitions), I think we should keep it and use `inline/{{ node.name }}.yaml`.

Reply:
Updated the path.